### PR TITLE
community_topics_workflow.md: change to conduct them on Forum

### PR DESCRIPTION
**Pull request by Andersson007**
To implement https://forum.ansible.com/t/draft-for-sc-voting-process-on-the-forum/1410

Removed all board mentions as it doesn't work:)

Closes: https://github.com/ansible-community/community-topics/issues/273

### CODE REVIEW

File: community_topics_workflow.md [hunk 1]
```diff
@@ -8,61 +8,71 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
...
+  - [ ] Click the `gear` button in the composer and select Build Poll.
+  - [ ] Click the `gear` in the Poll Builder for advanced mode.
+  - [ ] Set up the options (generally this will be Single Choice but other poll types can be used).
+  - [ ] Title it "Steering Committee vote" and "Limit voting" to the `Steering Committee`.
+  - [ ] Do not set the close date because this cannot be changed later.
+  - [ ] Results should be "Always Visible" unless there is some good reason for the SC votes not to be public.
+  - [ ] Submit the poll (the BBcode will appear in the post) and then repeat the above for the second poll.
+    - [ ] Title should be "Community vote".
+    - [ ] No group limitation.
+    - [ ] Results "Staff only" (i.e. an anonymous vote, but admins can check if there are concerns).
```
Comment (gotmax23) [inline community_topics_workflow.md:50]: Should we keep all votes like they were before we adopted the forum?

**Comment by gotmax23** [comment]
Thanks, @Andersson007 and @GregSutcliffe, for picking this up!

File: community_topics_workflow.md (hunk 1 above)
Comment (Andersson007) [reply community_topics_workflow.md:50]: @gotmax23 i'm not sure i understand though I'm for keeping them as they were. Please suggest the wording change

Comment (gotmax23) [reply community_topics_workflow.md:50]: Sorry, I was missing a word. I meant keep all votes public instead of making community votes anonymous.

Comment (Andersson007) [reply community_topics_workflow.md:50]: @gotmax23 ah, yeah, i would prefer to see them public too.
@GregSutcliffe do the formulations above imply community votes to be private? Is there any rationale for that and can we just remove the point containing "Staff only"?

**Comment by GregSutcliffe** [comment]
My rationale is that we may get more people to vote that way - some people prefer a veil of privacy. If there's any concern over sockpuppeting or flooding, the admins can still check who voted.

I'm not against making it public if you prefer though, it's your process ;)

File: community_topics_workflow.md (hunk 1 above)
Comment (Andersson007) [inline community_topics_workflow.md:50]:
```suggestion
```

**Comment by Andersson007** [comment]
@GregSutcliffe ok, thanks for explaining, makes sense but let's remove it for now and get back to it later if needed:)
@gotmax23 how about now?

**Comment by gotmax23** [comment]
Thanks @Andersson007! I'll take a closer look later today. In the meantime, it looks like there are some leftover references to the project board at the end of the doc that should be removed.

**Comment by Andersson007** [comment]
> Thanks @Andersson007! I'll take a closer look later today. In the meantime, it looks like there are some leftover references to the project board at the end of the doc that should be removed.

@gotmax23 removed, thanks, PTAL

**Review by gotmax23 — CHANGES_REQUESTED** [review]
LGTM, but we need a vote before merging this.

File: community_topics_workflow.md [hunk 2]
```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
...
+  - [ ] Creates a summary comment declaring the vote result.
 - [ ] The vote's result and the final decision are announced via the [Bullhorn](https://github.com/ansible/community/issues/546).
 
 ### Implementation stage
 
 - [ ] If the topic implies some actions (if it does not, just mark this as complete), the Committee person:
 
   - [ ] Assigns the topic to a person responsible for performing the actions.
-  - [ ] Add the `being_implemented` label to the topic.
+  - [ ] Add the `being_implemented` tag to the topic.
```
Comment (felixfontein) [inline community_topics_workflow.md:66]: Should this be changed to use a dash as well? Tags with underscores look strange to me.

File: community_topics_workflow.md [hunk 3]
```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
...
 
   - [ ] Comments on the topic that the work is done.
-  - [ ] Removes the `being_implemented` label.
-  - [ ] Add the `implemented` label.
+  - [ ] Removes the `being_implemented` tag.
+  - [ ] Add the `implemented` tag.
 - [ ] If the topic implies actions related to the future Ansible Community package releases (for example, a collection exclusion), the Committee person:
 
-  - [ ] Adds the `scheduled_for_future_release` label to the topic.
+  - [ ] Adds the `scheduled_for_future_release` tag to the topic.
```
Comment (felixfontein) [inline community_topics_workflow.md:74]: Same here.

**Review by felixfontein — COMMENTED** [review]
Besides that, LGTM.

File: community_topics_workflow.md (hunk 2 above)
Comment (Andersson007) [reply community_topics_workflow.md]: @felixfontein SGTM, changed, thanks, PTAL

File: community_topics_workflow.md [hunk 4]
```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
 
 The Workflow is a set of actions that need to be done successively within the corresponding time frames.
 
-After a topic is created, a Steering Committee member (hereinafter `Committee person`) copies the Workflow into the topic and, along the way, marks the checkboxes as complete to track the progress, see and plan the following actions needed to get the topic resolved within acceptable time.
-
 **Note:** If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
 
+## Creating a topic
+
+Any user can [created a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-meeting` to put it on the meeting agenda.
```
Comment (gotmax23) [inline community_topics_workflow.md:15]:
```suggestion
Any user can [create a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-meeting` to put it on the meeting agenda.
```

Comment (Andersson007) [reply community_topics_workflow.md]: @gotmax23 thanks for spotting, merged, PTAL

File: community_topics_workflow.md [hunk 5]
```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
 
 The Workflow is a set of actions that need to be done successively within the corresponding time frames.
 
-After a topic is created, a Steering Committee member (hereinafter `Committee person`) copies the Workflow into the topic and, along the way, marks the checkboxes as complete to track the progress, see and plan the following actions needed to get the topic resolved within acceptable time.
-
 **Note:** If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
 
+## Creating a topic
+
+Any user can [create a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-meeting` to put it on the meeting agenda.
```
Comment (gotmax23) [inline community_topics_workflow.md:15]:
```suggestion
Any user can [create a topic](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=steering-committee) tagged with `steering-committee` under the `Project Discussions` category in the [Ansible Forum](https://forum.ansible.com/). A steering committee member can tag the forum post with `community-wg-nextmtg` to put it on the meeting agenda.
```

File: community_topics_workflow.md [hunk 6]
```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
...
-- [ ] Adds the topic to the [Board](https://github.com/orgs/ansible-community/projects/2/views/1).
+- [ ] A Committee person checks the topic's content, asks the author/other persons to provide additional information if needed.
 
 ### Discussion stage
 
 - [ ] If the topic is ready to be discussed, the Committee person:
 
-  - [ ] Adds the `discussion_topic` label.
-  - [ ] Moves the topic to the `In Discussion` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+  - [ ] Adds the `community-meeting` tag if it needs to be discussed in the meeting.
```
Comment (gotmax23) [inline community_topics_workflow.md:29]:
```suggestion
  - [ ] Adds the `community-wg-nextmtg` tag if it needs to be discussed in the meeting.
```

**Comment by felixfontein** [comment]
Updated the PR according to https://github.com/ansible-community/community-topics/issues/273#issuecomment-1789513734.

**Review by gotmax23 — APPROVED** [review]

**Pull request merged**